modules/indexer/issues/dboptions.go

```diff
@@ -87,8 +85,10 @@ func ToSearchOptions(keyword string, opts *issues_model.IssuesOptions) *SearchOp
 	case "priority", "priorityrepo", "project-column-sorting":
 		// Unsupported sort type for search
 		searchOpt.SortBy = SortByUpdatedDesc
```

Comment:
Sorry, I misunderstood, plz ignore my comment above.
Your fix is right, it think it is better to fix it here.
Unsupported sort type should be same to default?

Reply:
No worries!

Given other reviews it was determined that we shouldn't change the default block. However, I did replace the only line of this case block with a `fallthrough` since it's the same as the `default` block.

Reply:
The latest version LGTM.